packages/mui-system/src/cssVars/createCssVarsProvider.spec.tsx

```diff
@@ -225,9 +215,9 @@ type Joy2ExtendedColorScheme = OverridableStringUnion<never, Joy2ColorSchemeOver
 type Joy2ColorScheme = 'light' | 'dark';
 
 interface Joy2Colors {
-  palette: {
-    primary: {
-      main: string;
+  palette?: {
+    primary?: {
+      main?: string;
```

Comment:
color input should be optional because it will be deep merged with the default one.